src/Illuminate/Database/DatabaseManager.php

```diff
@@ -245,6 +245,22 @@ public function reconnect($name = null)
         return $this->refreshPdoConnections($name);
     }
 
+    /**
+     * Set the database connection for the callback execution.
+     *
+     * @param  string  $name
+     * @param  callable  $callback
+     * @return void
+     */
+    public function usingConnection(string $name, callable $callback)
```

Comment:
No scalar types allowed.

Reply:
Oh sorry, I'll fix it

Reply:
Ah. It's already fixed in 60fd6002ba6f7bc2836de6d287d33f6804615e0a